### ISSUE TITLE
various places in the docs still refer to python2

### ISSUE THREAD

**Issue by rene-dev** [LinuxCNC/linuxcnc#1320]
they should be updated, if still relevant.
https://github.com/LinuxCNC/linuxcnc/search?q=python2

**Comment by petterreinholdtsen** [comment]
Definitely.  Who volunteers to provide a pull request?  Example code should be updated too.

**Comment by rmu75** [comment]
I think this is done?

**Comment by andypugh** [comment]
Mainly done. There is one referece to Python2.7 in the QTVCP Docs. 
https://github.com/LinuxCNC/linuxcnc/blob/master/docs/src/gui/qtvcp.adoc
